control-plane/connect-inject/webhook/consul_dataplane_sidecar.go

```diff
@@ -162,6 +162,15 @@ func (w *MeshWebhook) consulDataplaneSidecar(namespace corev1.Namespace, pod cor
 		container.VolumeMounts = append(container.VolumeMounts, volumeMounts...)
 	}
 
+	// Container Ports
+	ports, err := w.getContainerPorts(pod)
+	if err != nil {
+		return corev1.Container{}, err
+	}
+	if ports != nil {
+		container.Ports = ports
```

Comment:
Doesn't this incorrectly wipe out all the ports already set? Should it be an append?

On line 149:

		container.Ports = append(container.Ports, corev1.ContainerPort{
			Name:          fmt.Sprintf("%s-%d", "proxy-health", mpi.serviceIndex),
			ContainerPort: int32(constants.ProxyDefaultHealthPort + mpi.serviceIndex),

Reply:
Great catch! 🙇